timetracker/timetracker/urls.py

```diff
@@ -15,11 +15,24 @@
 """
 from django.conf.urls import include, url
 from django.contrib import admin
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpRequest
 
 
 def hello_world(request):
-    return HttpResponse('Hello World')
+	visitorName = request.GET.get("name", False)
+	responseString = ""
```

Comment:
In Python it's not necessary to have this here since you're assigning something to it in both your if and else.

Any variable created within a function gets the function level scoping so `responseString` will be available outside of the if/else.